src/main/java/duke/Event.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class contains the name, completion status and timing of the task.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Constructor for a Event object
+     * @param description The description of the task;
```

Comment:
Perhaps you should leave a line between the description and the @param line?

Reply:
Alright thanks